.github/workflows/compliance-monitoring.yml

```
@@ -0,0 +1,262 @@
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Check commit message compliance
+        id: commit-check
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2236:style:14:8: Use -n instead of ! -z

(shellcheck)